akvo/rsr/tests/rest/test_recipient_country.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
```

Comment:
Should we really test the API here? It's the RecipientCountry.iati_country() method that's the root problem.